### PR TITLE
RSS Button

### PR DESCRIPTION
**Pull request by AlphaT3ch**

### CODE REVIEW

**Comment by dbpokorny** [comment]
:+1:

**Comment by SpenserJ** [comment]
:+1:

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing :+1:`:+1:`, or :-1:`:-1:`.
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both :+1:up- and :-1:down-votes are disregarded.
Comments containing monkeys are disregarded. :monkey: :monkey_face: :see_no_evil: :hear_no_evil: :speak_no_evil:
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:*
 3e72e10cbaa1dc7dc74c6b1c11edbe8ef09f4c0c

File: data/views/layouts/main.hbs [hunk 1]
```diff
@@ -48,6 +48,12 @@
             position: relative;
             z-index: -2;
         }
+        .rss {
+            position: absolute;
+            right: 0px;
+            width: 300px;
+            background-color: #b0e0e6;
+    }
```
Comment (shalecraig) [inline data/views/layouts/main.hbs]: Please indent correctly.

**Comment by shalecraig** [comment]
sigh :-1:

**Comment by Prior99** [comment]
:+1: Because who needs correct indention? :see_no_evil:

**Comment by Prior99** [comment]
:-1: because of indention also, I repeat: Please delete this useless ideas.md file and use the issuetracker instead!

**Comment by plor** [comment]
:-1:

**Pull request closed without merging**